buildenvs/base.Dockerfile

```diff
@@ -0,0 +1,151 @@
+# FROM ${REG}/unikraft/kraftkit/gcc:${GCC_VERSION}-arm${GCC_SUFFIX}    AS gcc-arm
+# FROM ${REG}/unikraft/kraftkit/gcc:${GCC_VERSION}-arm64${GCC_SUFFIX}  AS gcc-arm64
+
+FROM ${REG}/unikraft/kraftkit/qemu:${QEMU_VERSION}                   AS qemu
+FROM ${REG}/unikraft/kraftkit/myself:${KRAFTKIT_VERSION}             AS kraft
+
+LABEL MAINTAINER="Alexander Jung <alexander.jung@neclab.eu>"
+LABEL MAINTAINER="Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>"
+
+# Copy the kraft tool
+COPY . /go/src/kraftkit.sh
+
+# Build the kraft tool - also installs it to the environment
+WORKDIR /go/src/kraftkit.sh
+
+RUN set -xe; \
+  make kraft;
+
+FROM debian:${DEBIAN_VERSION} AS kraftkit
+
+ARG GCC_PREFIX=x86_64-linux-gnu
+
+# Install kraftkit dependencies
+COPY --from=kraft /go/src/kraftkit.sh/dist/kraft /bin
+COPY --from=kraft /root/.config/kraftkit/config.yaml /root/.config/kraftkit/config.yaml
+COPY --from=kraft /root/.local/share/kraftkit /root/.local/share/kraftkit
+
+# Copy the GCC toolchain
+COPY --from=gcc-x86_64 /bin/                                         /bin
+COPY --from=gcc-x86_64 /lib/gcc/                                     /lib/gcc
+COPY --from=gcc-x86_64 /include/                                     /include
+COPY --from=gcc-x86_64 /x86_64-linux-gnu                             /x86_64-linux-gnu
+COPY --from=gcc-x86_64 /libexec/gcc/x86_64-linux-gnu/12.2.0/cc1      /libexec/gcc/x86_64-linux-gnu/12.2.0/cc1
+COPY --from=gcc-x86_64 /libexec/gcc/x86_64-linux-gnu/12.2.0/collect2 /libexec/gcc/x86_64-linux-gnu/12.2.0/collect2
+# COPY the others if needed
+
+# COPY --from=gcc-arm /bin/                                            /bin
+# COPY --from=gcc-arm /lib/gcc/                                        /lib/gcc
+# COPY --from=gcc-arm /include/                                        /include
+# COPY --from=gcc-arm /lib/gcc/                                        /lib/gcc
+# COPY --from=gcc-arm /arm-linux-gnueabihf                             /arm-linux-gnueabihf
+# COPY --from=gcc-arm /libexec/gcc/arm-linux-gnueabihf/12.2.0/cc1      /libexec/gcc/arm-linux-gnueabihf/12.2.0/cc1
+# COPY --from=gcc-arm /libexec/gcc/arm-linux-gnueabihf/12.2.0/collect2 /libexec/gcc/arm-linux-gnueabihf/12.2.0/collect2
+# COPY the others if needed
+
+# COPY --from=gcc-arm64 /bin/                                          /bin
+# COPY --from=gcc-arm64 /lib/gcc/                                      /lib/gcc
+# COPY --from=gcc-arm64 /include/                                      /include
+# COPY --from=gcc-arm64 /lib/gcc/                                      /lib/gcc
+# COPY --from=gcc-arm64 /aarch64-linux-gnu/                            /aarch64-linux-gnu
+# COPY --from=gcc-arm64 /libexec/gcc/aarch64-linux-gnu/12.2.0/cc1      /libexec/gcc/aarch64-linux-gnu/12.2.0/cc1
+# COPY --from=gcc-arm64 /libexec/gcc/aarch64-linux-gnu/12.2.0/collect2 /libexec/gcc/aarch64-linux-gnu/12.2.0/collect2
+# COPY the others if needed
```

Comment:
Eventually these will be added, why aren't they included atm? I forgot

Reply:
well, first of all they are not available

Also, we are currently not compiling `gcc` for those arches

Also not sure how well kraftkit works with `arm`

I suggest we leave them like this for now